src/App.css

```diff
@@ -27,8 +27,18 @@
 .addColorInputBtnContainer {
   display: flex;
   margin: 0.5rem 0 2rem;
+  column-gap: 0.5rem;
+}
+.addColorInputBtnContainer .addColorBtn {
+    background-color: var(--text);
+}
+.addColorInputBtnContainer .resetBtn {
+  background-color: var(--grey);
+}
+.addColorInputBtnContainer img {
```

Comment:
Please use react icons instead of the image

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix